mypy/checkexpr.py

```diff
@@ -1608,13 +1608,17 @@ def analyze_super(self, e: SuperExpr, is_lvalue: bool) -> Type:
                         return AnyType()
                     if not self.chk.in_checked_function():
                         return AnyType()
-                    selftype = self.chk.function_stack[-1].arguments[0].variable.type
-                    return analyze_member_access(name=e.name, typ=fill_typevars(e.info), node=e,
+                    # fill_typevars(e.info) erases type variables
```

Comment:
This comment isn't right. `fill_typevars` adds type variables, erasure means something like stripping away type variables or replacing them with the upper bound / `Any`, which isn't happening here. If I have a class define as `class C(Generic[T]):` then `fill_typevars` will create a type `C[T]` when called with the type info for `C` as an argument.